src/custom_hooks/copyright_checker.py

```diff
@@ -204,10 +206,11 @@ def copyright_checker(filenames: list[str], owner: str, update: bool) -> int:
     Run copyright check on each file.
     """
     result = 0
-    repo = git.Repo(".")
+    repo = git.Repo(os.getenv("GIT_REPO_DIR", "."))
```

Comment:
I am not a fan of nested-hooks since I think they are very messy and hard to debug so don't think this is something that I actually want to fix. Not sure it's even an issue. Why can't you move your use case to the top level and use a regex to include / exclude the files that you don't want this hook to run?

Reply:
Agreed that this doesn't seem like a great fit.

It's not clear why this change would be needed. Maybe if you shared the implementation of the other code that sets this environment variable it would be clearer

Further copyrights should probably apply to an entire repository instead of a specific directory since the entire organization and repository have rules around that.